### ISSUE TITLE
Spurious backup operation on restart

### ISSUE THREAD

**Issue by MetRonnie** [cylc/cylc-flow#4636]
- `cylc play myflow` and stop it before it finishes
```
INFO - Extracting job.sh to ~/cylc-run/myflow/.service/etc/job.sh
...
```

So far so good. Now:
- `cylc play myflow` again to restart
```diff
 INFO - Extracting job.sh to ~/cylc-run/myflow/.service/etc/job.sh
+WARNING - Replacing an existing cylc-tutorials folder which will be copied to ~/cylc-run/myflow/.service/etc/job.sh.20220131T175412Z
...
```

Assigning @wxtim as this looks to be related to the get-resources change

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by oliver-sanders** [comment]
Is this actually a bug?

**Comment by MetRonnie** [comment]
It shouldn't be doing that, so yes?

**Comment by oliver-sanders** [comment]
I think this is a low priority "could be better".

**Comment by MetRonnie** [comment]
Imho the fix is so small it's worth getting out of the way now
